src/Native/System.IO.Compression.Native/ZLib-1.2.3/ChangeLog

```diff
@@ -0,0 +1,855 @@
+
+                ChangeLog file for zlib
+
+Changes in 1.2.3 (18 July 2005)
```

Comment:
For the record, I agree. I have no plans on leaving it at 1.2.3; I wanted something up sooner rather than later to unblock https://github.com/dotnet/corefx/issues/3986 and get this out in the open. As you said, just a step in the journey :)

> It'd be great to use newer source at the very least

This should be trivial. I'll go ahead and put in an OSS request for 1.2.8 (latest). Replacing 1.2.3 with 1.2.8 will be a copy-paste job and maybe some variable modifications in the CMakeList. I'll know better when approval comes. We can hold off on this until 1.2.8 approval comes, if you'd prefer.

>ideally just use some official binary distribution instead

We can still do this. Adler uploads a windows x86/x64 package to NuGet that works out-of-the-box with System.IO.Compression. This locks us out of making any native modifications, but there's always the argument that we can submit those as a PR to the zlib github if we want them badly enough. Same goes for support for other architectures. Not sure if we could do that with the Intel mods that bjjones was talking about adding, but I haven't looked into that.